src/utils/parser.js

```diff
@@ -181,6 +198,31 @@ function getSymbols(source: SourceText): SymbolDeclarations {
   return symbols;
 }
 
+function getExpression(source: SourceText, token: string, location: Location) {
+  let expression = null;
+  const ast = getAst(source);
+
+  if (isEmpty(ast)) {
+    return;
+  }
+
+  traverse(ast, {
+    enter(path) {
+      const node = path.node;
+      if (node.type === "MemberExpression" && node.property.name === token
```

Comment:
Utilizing babel-types across the parser would be a good idea. I think there's a place in the function parsing helpers where I could do this too.